src/main.rs

```diff
@@ -1,511 +1,52 @@
+mod api;
 mod args;
+mod lnd;
+pub mod scheduler;
 
-use std::collections::HashMap;
-use std::convert::TryInto;
-use std::fmt;
-use std::sync::{Arc, Mutex};
-
-use args::ArgError;
-use bip78::receiver::*;
-use bitcoin::util::address::Address;
-use bitcoin::util::psbt::PartiallySignedTransaction;
-use bitcoin::{Script, TxOut};
-use hyper::service::{make_service_fn, service_fn};
-use hyper::{Body, Method, Request, Response, Server, StatusCode};
-use ln_types::P2PAddress;
+use lnd::LndClient;
+use scheduler::ScheduledPayJoin;
 
 use crate::args::parse_args;
+use crate::scheduler::Scheduler;
 
 #[macro_use]
 extern crate serde_derive;
 extern crate configure_me;
 
 configure_me::include_config!();
 
-#[cfg(not(feature = "test_paths"))]
```

Comment:
This was removed and not replaced. Please put that change in its own commit to document the change for review